pkg/runtime/runtime.go

```diff
@@ -275,14 +270,8 @@ func (a *DaprRuntime) initRuntime(opts *runtimeOpts) error {
 	}
 
 	a.flushOutstandingComponents()
```

Comment:
This call should make sure the entire components are loaded and initialized. So I do not think the issue is not related to L266 go routine.

cc/ @wcs1only